tests/ignite/engine/test_create_supervised.py

```diff
@@ -103,7 +108,7 @@ def test_create_supervised_trainer_on_cuda():
     assert model.bias.item() == approx(0.8)
 
 
-@pytest.mark.tpu
+@pytest.mark.skipif(not has_xla, reason="Skip if no TPU")
```

Comment:
We still need to mark with `@pytest.mark.tpu` as here : https://github.com/pytorch/ignite/blob/bf052c9435a53ec324a7f9a1bcfa1ff075a6fd73/tests/ignite/metrics/test_accumulation.py#L375